oneview/resource_logical_interconnect_group.go

```diff
@@ -151,7 +146,6 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 						"lacp_timer": {
 							Type:     schema.TypeString,
 							Optional: true,
-							Default:  "Short",
```

Comment:
As discussed in meeting about ethernet network we will be keeping it optional only.

Reply:
what i mean to say in case of ethernet network default value would be short so OV will set it to Auto but in terraform config file if its set as lacp_timer = "" then idempotency issue may come